fiftyone/plugins/context.py

```
@@ -23,6 +25,89 @@
+
+    # Remove any remaining non-alphanumeric characters except underscore
+    name = re.sub(r"[^a-z0-9_]", "", name)
+
+    # Ensure doesn't start with a number
+    if name and name[0].isdigit():
+        name = "_" + name
+
+    return name
+
+
+def _get_plugin_module_name(plugin_name, fallback_name=None):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_get_plugin_module_name`

Add return type annotation: `str`

(ANN202)